src/functions/New-Session.ps1

```diff
@@ -56,14 +57,22 @@
         # Specify Access Token
         # Bypasses requesting a token from Secret Server
         [Parameter(ParameterSetName = 'sdk', Mandatory)]
-        $AccessToken
+        $AccessToken,
+
+        # Uses the credentials of the current user to create session.
+        [Parameter(ParameterSetName = 'iwa', Mandatory)]
+        [switch]
+        $UseDefaultCredentials
     )
 
     begin {
         $newTssParams = $PSBoundParameters
         $invokeParams = @{ }
 
         $outputTssSession = [TssSession]::new()
+        if ($UseDefaultCredentials.IsPresent) {
```

Comment:
Please use `$newTssParams.ContainsKey('UseDefaultCredentials')`

Reply:
in 7614719